isce2_topsapp/delivery_prep.py

```diff
@@ -115,26 +124,30 @@ def format_metadata(nc_path: Path,
     b_perp = read_baseline_perp(nc_path).mean()
 
     metadata = {}
-    metadata.update({"ogr_bbox": geojson['coordinates'],
+    # get 4 corners of bounding box of the geometry; default is 5 returning
+    # to start point
+    ogr_bbox = all_metadata['intersection_geo'].envelope.exterior.coords[:4]
+    metadata.update({"ogr_bbox": ogr_bbox,
                      "reference_scenes": all_metadata['reference_scenes'],
                      "secondary_scenes": all_metadata['secondary_scenes'],
-                     "sensing_start": ref_props['startTime'],
-                     "sensing_stop": ref_props['stopTime'],
-                     "orbit_number": [ref_props['orbit'], sec_props['orbit']],
-                     "platform": ref_props['platform'],
+                     "sensing_start": f'{ref_props["startTime"]}Z',
+                     "sensing_stop": f'{ref_props["stopTime"]}Z',
+                     "orbit_number": [int(ref_props['orbit']),
+                                      int(sec_props['orbit'])],
+                     "platform": [ref_props['platform'], sec_props['platform']],
                      "beam_mode": ref_props['beamModeType'],
                      "orbit_direction": ref_props['flightDirection'].lower(),
                      "dataset_type": 'slc',
                      "product_type": 'interferogram',
                      "polarization": "HH",
```

Comment:
Possibly out of scope for this PR, but I'd expect polarization to be determined from the input SLC(s) rather than being hardcoded.

Reply:
Wow - that's a very good observation.

I actually don't know precisely - I believe it's a topsapp default. I just pass the entire zip file as here: https://github.com/ACCESS-Cloud-Based-InSAR/DockerizedTopsApp/blob/dev/isce2_topsapp/templates/topsapp_template.xml

And then the ISCE2 topsapp I believe just uses HH.